soc/arm/nxp_kinetis/kv5x/dts_fixup.h

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (c) 2019 SEAL AG
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* SoC level DTS fixup file */
+#define DT_NUM_IRQ_PRIO_BITS		DT_ARM_V7M_NVIC_E000E100_ARM_NUM_IRQ_PRIORITY_BITS
+
+#define DT_SIM_NAME			DT_NXP_KINETIS_SIM_40047000_LABEL
+#ifdef DT_NXP_KINETIS_SIM_40047000_CLKOUT_DIVIDER
+#define DT_SIM_CLKOUT_DIVIDER		DT_NXP_KINETIS_SIM_40047000_CLKOUT_DIVIDER
+#endif /* DT_NXP_KINETIS_SIM_40047000_CLKOUT_DIVIDER */
+#ifdef DT_NXP_KINETIS_SIM_40047000_CLKOUT_SOURCE
+#define DT_SIM_CLKOUT_SOURCE		DT_NXP_KINETIS_SIM_40047000_CLKOUT_SOURCE
+#endif /* DT_NXP_KINETIS_SIM_40047000_CLKOUT_SOURCE */
+
+#define DT_FLASH_DEV_BASE_ADDRESS	DT_NXP_KINETIS_FTFE_40020000_BASE_ADDRESS
+#define DT_FLASH_DEV_NAME		DT_NXP_KINETIS_FTFE_40020000_LABEL
+
+#define DT_ADC_0_BASE_ADDRESS		DT_NXP_KINETIS_ADC16_4003B000_BASE_ADDRESS
+#define DT_ADC_0_IRQ			DT_NXP_KINETIS_ADC16_4003B000_IRQ_0
+#define DT_ADC_0_IRQ_PRI		DT_NXP_KINETIS_ADC16_4003B000_IRQ_0_PRIORITY
+#define DT_ADC_0_NAME			DT_NXP_KINETIS_ADC16_4003B000_LABEL
+
+#define DT_FTM_1_BASE_ADDRESS		DT_NXP_KINETIS_FTM_40038000_BASE_ADDRESS
```

Comment:
This should start at DT_FTM_0 and go to 3.

Reply:
Doesn't that create a mismatch between the FTM driver and the device tree, because with CONFIG_PWM_(x)=y, the FTM driver needs DT_FTM_(x)_BASE_ADDRESS (and the rest) to be defined? I presume the pwm alias numbers in device tree should correspond to CONFIG_PWM_{0,1,2...}